src/commands/delete.ts

```diff
@@ -53,26 +53,26 @@ function getItemToDelete(contextItem: TektonNode, selectedItems: TektonNode[]):
 
 async function doDelete(items: TektonNode[], toRefresh: Refreshable, commandId?: string): Promise<void | string> {
   if (items) {
-    const taskRunList = await getTaskRunResourceList();
     let message: string;
-    let refResource = true;
+    let hasAnyReference = true;
```

Comment:
maybe you should init it with false, otherwise the meaning is not correct with the naming. I mean, if you look at below you have
```
if (hasAnyReference) {
        message = `Do you want to delete ${toDelete.size} items?`;
      } else {
        message = `Do you want to delete ${toDelete.size} items?. You have selected Task or ClusterTask which is being used in some other resource.`;
``` 
which is wrong. It should be the opposite.

```
if (!hasAnyReference) {
        message = `Do you want to delete ${toDelete.size} items?`;
      } else {
        message = `Do you want to delete ${toDelete.size} items?. You have selected Task or ClusterTask which is being used in some other resource.`;
```